bin/seed-from-staging

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
+
+require_relative '../deployment'
+require 'cdo/rake_utils'
+
+Dir.chdir(dashboard_dir) do
+  staging_dir = '../../staging'
```

Comment:
This code expects there to be a staging directory that is a git repository. Can you add a comment to this script describing the prerequisites here? If that directory is not present, will this simply error out with an appropriate message?

Reply:
Yes the script would previously exit with an exception if the directory was not found. I have a better error message now.